src/components/pages/Chat/AddRoomItem.tsx

```diff
@@ -179,21 +188,13 @@ function UnstyledAddRoomItem(props: Props) {
                         </ModalHeader>
                         <div>
                             <Subheading>Name</Subheading>
-                            <input
-                                disabled={true}
-                                placeholder="random-name-abc"
-                            />
+                            <input defaultValue={roomName} />
```

Comment:
This does nothing. `value` and `onChange` are missing thus rooms get created with random names.

Reply:
`/new roomname` created a room named "roomname", I confirm that.

Reply:
You're totally right, just added the `onChange` handler to update `setRoomName` and it's working alright now